examples/rules/domain.yml

```diff
@@ -0,0 +1,80 @@
+intents:
+- activate_q_form
+- inform
+- explain
+- stopp
+- ask_possibilities
+- faq
+- affirm
+- deny
+- greet
+- switch_faq
+- nlu_fallback
+
+entities:
+- some_slot
+
+slots:
+  some_slot:
+    type: unfeaturized
+  detailed_faq:
+    type: bool
+
+actions:
+- utter_explain_some_slot
+- action_stop_q_form
+- utter_list_possibilities
+- utter_faq
+- utter_ask_did_help
+- utter_continue
+- utter_detailed_faq
+- utter_ask_stop
+- utter_stop
+- utter_greet
+- action_switch_faq
+- utter_did_you_mean
+- action_validate_loop_q_form
+
+forms:
+- loop_q_form:
+    some_slot:
+    # Slot mappings can be defined in the domain.
+    # There will be an option in the future to overwrite this from the custom action
+    # server.
+    # The slot mappings follow the same syntax as currently in the SDK implementation
+    - type: from_entity
+      entity: some_slot
+#    some_slot:
+#    - type: from_text
```

Comment:
minor: can this be removed?

Reply:
I had it there as an example, will add some explainatory comments